scripts/common/sonic-utilities-build/build.sh

```diff
@@ -17,6 +17,8 @@ sudo pip3 install buildimage/target/python-wheels/sonic_yang_models-1.0-py3-none
 sudo dpkg -i buildimage/target/debs/buster/libyang_1.0.73_amd64.deb
 sudo dpkg -i buildimage/target/debs/buster/libyang-cpp_1.0.73_amd64.deb
 sudo dpkg -i buildimage/target/debs/buster/python2-yang_1.0.73_amd64.deb
+sudo dpkg -i buildimage/target/debs/buster/libswsscommon_1.0.0_amd64.deb
+sudo dpkg -i buildimage/target/debs/buster/python-swsscommon_1.0.0_amd64.deb
```

Comment:
install python3 version as well?

Reply:
Resolved

---
In reply to: [518270554](https://github.com/Azure/sonic-build-tools/pull/181#discussion_r518270554) [](ancestors = 518270554)